onnxscript/function_libs/torch_lib/graph_building.py

```diff
@@ -925,6 +938,18 @@ def to_function_proto(self, opset_version: int, function_name: str) -> onnx.Func
 
         onnx_model = onnx.load_from_string(proto)
 
+        unique_custom_domains = self.fetch_opset_imports()
+
+        while len(onnx_model.opset_import) > 0:
```

Comment:
Is there a way to use a for loop for this to avoid unintentional infinite loops?

Reply:
certainly